app/models/infrastructure.rb

```diff
@@ -121,4 +129,19 @@ def allow_delete?
       'DELETE_ERROR'
     ].include?(self.provisioning_status) && self.status == 'INACTIVE'
   end
+
```

Comment:
what are these methods, do you have tests for this?

Reply:
I believe these methods were there before we made this PR.